doc/api/fs.md

```diff
@@ -2529,6 +2529,18 @@ changes:
 Asynchronous rename(2). No arguments other than a possible exception are given
 to the completion callback.
 
+This function will take the `oldPath` of a file and rename it to the
+pathname provided as `newPath`.
+
+Example renames 'firstfile.txt' to 'secondfile.txt':
```

Comment:
Again, telling people that an example follows is unnecessary. Just provide the example. Remove this line.